KeyboardShortcutsExample/ContentView.swift

```diff
@@ -4,9 +4,79 @@ import KeyboardShortcuts
 extension KeyboardShortcuts.Name {
 	static let testShortcut1 = Self("testShortcut1")
 	static let testShortcut2 = Self("testShortcut2")
+	static let testShortcut3 = Self("testShortcut3")
+	static let testShortcut4 = Self("testShortcut4")
 }
 
-struct ContentView: View {
+struct Shortcut {
+	let name: KeyboardShortcuts.Name
+	let label: String
+}
+
+final class PickerShortcutViewModel: ObservableObject {
+	@Published var isPressed = false
+	@Published var selectedState = 0 {
+		didSet {
+			KeyboardShortcuts.disable(shortcuts[oldValue].name)
+			setShortcutEvent(name: shortcuts[selectedState].name)
+		}
+	}
+
+	var selectedLabel: Binding<String> {
+		Binding(
+			get: {
+				self.shortcuts[self.selectedState].label
+			},
+			set: { label in
+				self.selectedState = self.shortcuts.firstIndex { $0.label == label } ?? 0
+			}
+		)
+	}
+	
+	var shortcuts = [
+		Shortcut(name: .testShortcut3, label: "Shortcut3"),
+		Shortcut(name: .testShortcut4, label: "Shortcut4")
+	]
+
+	init() {
+		setShortcutEvent(name: shortcuts[selectedState].name)
+	}
+
+	func setShortcutEvent(name: KeyboardShortcuts.Name) {
+		KeyboardShortcuts.onKeyDown(for: name) {
+			self.isPressed = true
+		}
+
+		KeyboardShortcuts.onKeyUp(for: name) {
+			self.isPressed = false
+		}
+	}
+}
+
+struct PickerShortcut: View {
+	@ObservedObject private var viewModel = PickerShortcutViewModel()
+
+	var body: some View {
+		VStack {
```

Comment:
I think this should have a header that says "Dynamic recorder" to make it clear that the example shows off how to use dynamic recorders.